provider-springmvc/src/main/java/io/servicecomb/provider/springmvc/reference/RestTemplateWrapper.java

```diff
@@ -27,26 +29,43 @@
 import org.springframework.web.client.RestClientException;
 import org.springframework.web.client.RestTemplate;
 
-import io.servicecomb.common.rest.RestConst;
-
 /**
  * 用于同时支持cse调用和非cse调用
  */
 public class RestTemplateWrapper extends RestTemplate {
-    private static RestTemplate cseRestTemplate = new CseRestTemplate();
+    private static List<AcceptableRestTemplate> acceptableRestTemplates = new ArrayList<>();
```

Comment:
and i don't understand why these methods need to be static. we have way too many static classes/methods. this kind of programming model creates tons of tight coupling.